includes/rest-api/Controllers/Version2/class-wc-rest-orders-v2-controller.php

```diff
@@ -387,10 +384,10 @@ protected function get_formatted_item_data( $order ) {
 	public function prepare_object_for_response( $object, $request ) {
 		$this->request       = $request;
 		$this->request['dp'] = is_null( $this->request['dp'] ) ? wc_get_price_decimals() : absint( $this->request['dp'] );
-		$data                = $this->get_formatted_item_data( $object, $request );
-		$context             = ! empty( $request['context'] ) ? $request['context'] : 'view';
+		$request['context']  = isset( $request['context'] ) ? $request['context'] : 'view';
```

Comment:
Can you maybe expand a bit on 
1. why you switched from `! empty()` to `isset`? (I think we have other places in code where we check with `! empty()`, we might need to update those, too? 
2. why you decided to set the context on the `$request` object here?

Reply:
> why you switched from ! empty() to isset? (I think we have other places in code where we check with ! empty(), we might need to update those, too?

This is an innocent change, I thought it was an easier read given the tertiary condition instead of using `! empty`. I looked up difference between is_empty and isset, and there is none if the possible values are always truthy, which is the case here with values `view` or `edit`. So these are synonymous. If you think we should be consistent here I can change to use `! empty`.

> why you decided to set the context on the $request object here?

We are using `$request` or `$this->request` in `get_formatted_item_data` which we call in next line so if the context is set in request, we can read it from there instead of having to pass another parameter.

We are already using the `$this->request` object in `get_formatted_item_data` [method for getting dp precision](https://github.com/woocommerce/woocommerce/blame/master/includes/rest-api/Controllers/Version2/class-wc-rest-orders-v2-controller.php#L165), so passing `context` in request object and then using it in `get_formatted_item_data` does not add any additional coupling I think.

Reply:
> ! empty() vs isset

I was just afraid of repeating the null vs empty string situation again. I agree it's easier to read with `isset`. I was just wondering why all the other places in API code consistently use `! empty`. Let's maybe switch it for the sake of consistency? 

> $request['context']

👍  Sounds good.

Reply:
>  Let's maybe switch it for the sake of consistency?

👍 done in https://github.com/woocommerce/woocommerce/pull/28677/commits/3a7e3fda3467d62c60c77dc93318377ae2088f45